src/nunavut/lang/c/templates/_canard_set.j2

```diff
@@ -0,0 +1 @@
+out_buffer, offset + length, in_instance.{{ field.name }}[i]
```

Comment:
Use [Jinja macros](https://jinja.palletsprojects.com/en/2.11.x/templates/#macros) for DRY instead of including this file.

Reply:
Hah, in an uncommitted comment:

```
{#- TODO make a macro -#}
```